### PR TITLE
fix: error_parsing

### PR DESCRIPTION
**Pull request by AAfghahi**
### SUMMARY

There was SQL information in the logs that we were sending, this fixes the error message to make sure that it does not happen. 

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset/reports/commands/execute.py [hunk 1]
```diff
@@ -16,6 +16,7 @@
 # under the License.
 import json
 import logging
+import re
```
Comment (Antonio-RiveroMartnez) [inline superset/reports/commands/execute.py:19]: Are we using it somehow in this file?

Comment (AAfghahi) [reply superset/reports/commands/execute.py:19]: no this was leftover! Thank you!

Comment (AAfghahi) [reply superset/reports/commands/execute.py]: fixed

**Review by Antonio-RiveroMartnez — APPROVED** [review]
LGTM!

**Review by eschutho — APPROVED** [review]

**Pull request merged**